src/cogent3/app/data_store_new.py

```diff
@@ -426,14 +426,11 @@ def summary_not_completed(self) -> TabularType:
         return Table(header=header, data=rows, title="not completed records")
 
     def __repr__(self):
-        notcompleted_path = Path(self.source / _NOT_COMPLETED_TABLE)
-        if not notcompleted_path.exists():
-            return "This isn't a results directory"
         num_completed = len(self)
         num_not_completed = len(self.not_completed)
         name = self.__class__.__name__
         sample = f"{str(list(self[:3]))[:-1]}..." if num_completed > 3 else list(self)
```

Comment:
not sure why slicing twice ([:3] then [:-1]), also change to being a `repr()` rather than string
```python
sample = f"{list(self[:2])}..." if num_completed > 3 else list(self)
```